### DISCUSSION TITLE
Separate Taskbars on different Workspaces

### DISCUSSION THREAD

**Discussion by tsposato** [linuxmint/discussions#607] [Ideas]
Currently I mainly utilise 2 workspaces, 1 general and 1 for work.
The applications I use are very different across the 2 workspaces, and I would like to pin different applications to the taskbar for each individual workspace.
I don't believe this is currently possible in any automated way, potential it could be done with panels and manually hiding/showing them for each workspace...
I didn't see any similar discussion for this sort of feature, or if it would be possible, but it would be a great addition to cinnamon if it was possible!

**Comment by klangman** [comment]
I don't recall if it's possible in the default Window List, but I added such a feature in my CassiaWindowList Applet (https://cinnamon-spices.linuxmint.com/applets/view/372).  The option is in the applet configuration under the General tab called "Pinning of window list buttons" and it should be set to "Unique for each workspace".

**Reply by tsposato** [reply]
If I use the CassiaWindowList as my 'Taskbar' do I then need to disable the default one?
Currently I have both shown...

Ahh I needed to remove the Grouped Window list, which I have now done.
Is there a way to default the CassiaWindowList so that it is left aligned?

I found it, I think this is exactly what I was looking for... thanks - I'll play around with it a bit more!
